frontend/www/js/omegaup/group/edit.ts

```diff
@@ -298,68 +298,92 @@ OmegaUp.on('ready', () => {
       }),
     );
   }
+});
 
-  function generatePassword(humanReadable: boolean): string {
-    if (humanReadable) {
-      const words = {
-        es: [
-          'Loro',
-          'Perro',
-          'Pollo',
-          'Lagarto',
-          'Gato',
-          'Toro',
-          'Vaca',
-          'Sapo',
-          'Oso',
-          'Zorro',
-        ],
-        en: [
-          'Parrot',
-          'Dog',
-          'Chicken',
-          'Lizard',
-          'Cat',
-          'Bull',
-          'Cow',
-          'Frog',
-          'Bear',
-          'Fox',
-        ],
-      };
-      const itemsNumber = 3;
-      const langWords = T.lang === 'es' ? words.es : words.en;
-      let password = '';
-      for (let i = 0; i < itemsNumber; i++) {
-        password += langWords[Math.floor(Math.random() * langWords.length)];
-      }
-      for (let i = 0; i < itemsNumber; i++) {
-        password += Math.floor(Math.random() * 10); // 3 random numbers
-      }
-      return password;
-    }
+export function generatePassword(humanReadable: boolean): string {
+  if (humanReadable) {
+    const words = {
+      es: [
+        'Loro',
+        'Perro',
+        'Pollo',
+        'Lagarto',
+        'Gato',
+        'Toro',
+        'Vaca',
+        'Sapo',
+        'Oso',
+        'Zorro',
+      ],
+      en: [
+        'Parrot',
+        'Dog',
+        'Chicken',
+        'Lizard',
+        'Cat',
+        'Bull',
+        'Cow',
+        'Frog',
+        'Bear',
+        'Fox',
+      ],
+      pt: [
+        'Papagaio',
+        'Cão',
+        'Frango',
+        'Lagarto',
+        'Gato',
+        'Touro',
+        'Vaca',
+        'Sapo',
+        'Suportar',
```

Comment:
```suggestion
        'Urso',
```

jajajaja, Google Translate tiene un bug horrible! "Suportar" es la traducción de "Bear" en su forma de [verbo](https://www.merriam-webster.com/dictionary/bear#dictionary-entry-2) y sale si traduces "oso" de español a portugués!

Reply:
Jajaja, ni me fijé, mala mía